device/device.go

```diff
@@ -0,0 +1,110 @@
+package device
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/device/deviceType"
+	"github.com/josephroberts/edge-node-manager/radio/bluetooth"
+	"github.com/josephroberts/edge-node-manager/radio/radioType"
+	"github.com/josephroberts/edge-node-manager/radio/wifi"
+	"github.com/josephroberts/edge-node-manager/radio/zigbee"
+)
+
+type Type struct {
```

Comment:
I really don't like that you have `device.Type` and `deviceType.Type` and they're different. It's extremely confusing. I really don't know what you're trying to do here?

If you have a `deviceType` package that should handle everything relating to device types. Again the fact it's totally uncommented doesn't help either, when you have odd duplication like this you should try to explain why.